flac.go

```diff
@@ -94,41 +121,41 @@ var id3Signature = []byte("ID3")
 // stream, and parses the StreamInfo metadata block. It returns a boolean value
 // which specifies if the StreamInfo block was the last metadata block of the
 // FLAC stream.
-func (stream *Stream) parseStreamInfo() (isLast bool, err error) {
+func (stream *Stream) parseStreamInfo() (block *meta.Block, err error) {
```

Comment:
> thought: When I think about it, stream's parseStreamInfo has a name conflict with block's parseStreamInfo, maybe we should rename one of them. What do you think @mewmew?

Both are unexported so we can rename without changing exported API. Personally, I think it is fine as is, but would also be fine with changing the name of one to avoid confusion. The are distinct by one being a function, the other a method, so the call sites will be different. If we do change the name, e.g. of the method `parseStreamInfo` the names of other methods parsing blocks should be updated for consistency.